okta/okta.go

```diff
@@ -150,6 +150,19 @@ func (c *Client) GetConfig() *config {
 	return c.config
 }
 
+func (c *Client) SetConfig(conf ...ConfigSetter) (err error) {
```

Comment:
I'm not sure sure the best way to document this is. Just on the public method? Also in the readme?